client/package.json

```diff
@@ -18,9 +18,10 @@
     "typings": "./lib/index",
     "dependencies": {
         "glob-to-regexp": "^0.3.0",
-        "vscode-jsonrpc": "^5.0.1",
-        "vscode-languageclient": "^6.1.3",
-        "vscode-uri": "^2.1.1"
+        "vscode-jsonrpc": "6.0.0-next.7",
+        "vscode-languageclient": "7.0.0-next.12",
```

Comment:
Can you give a short explanation on why we use the unreleased versions of these packages? Do we rely on features that are not yet available with the main releases?

Reply:
The last 6.x of `vscode-languageclient` and the last 5.x of vscode-jsonrpc are 8 months old, so just after the release of monaco 0.20.0

Since everything is coming from the vscode stack, I thouth it's a good idea to keep all packages at the same "update level", but maybe I'm wrong?

Reply:
Oh I see, I'm fine with that.